Cargo.toml

```diff
@@ -11,6 +11,8 @@ description = "A LoRa physical layer implementation enabling utilization of a ra
 [dependencies]
 defmt = { version = "0.3" }
 log = { version = "0.4" }
+#lora-modulation = { version = "0.1.1" }
+lora-modulation = { git = "https://github.com/ivajloip/rust-lorawan.git", branch = "lthiery/impl-sfbw" }
```

Comment:
@ceekdee will have to decide on this but it feels like this crate should not depend on a lorawan crate. For p2p purposes this requirement is a bit backward.

Reply:
technically, the crate is "lora-modulation" and happens to be hosted in the rust-lorawan repo. As you can see above, the rust-lorawan repo isn't in the name when you pull it from crates.io

Reply:
The lora-modulation crate will be used by lora-phy version 2 once it is in final form on crates.io.